neocore/KeyPair.py

```diff
@@ -63,10 +63,6 @@ def __init__(self, priv_key):
             except Exception as e:
                 raise Exception("Could not determine public key")
 
-        elif length == 64 or length == 72:
```

Comment:
this code can never be reached because of the following check happing first
```
if length != 32 and length != 96 and length != 104:
            raise ValueError("Invalid private key")
```